test.py

```diff
@@ -2,3 +2,5 @@
 
 for number in numbers: 
     print(number)
+
+print("hello there")
```

Comment:
Hey @christinastrs this feels wrong because you have no new line at the end, could you please fix this?